packages/flutter/lib/src/rendering/image.dart

```diff
@@ -114,6 +116,20 @@ class RenderImage extends RenderBox {
     markNeedsPaint();
   }
 
+  /// Used to combine [color] with this image.
+  ///
+  /// The default is [BlendMode.srcIn]. In terms of the blend mode, [color] is
+  /// the source and this image is the destination.
+  BlendMode get colorBlendMode => _colorBlendMode;
```

Comment:
Please point to `[BlendMode]` for a discussion of the values and their meanings.